classes/RightSignature.php

```diff
@@ -77,9 +77,9 @@ public function prepackageTemplate($templateGuid, $callbackUrl=null)
 	 * @param int $documentGuid document GUID
 	 * @return RightSignature\Document
 	 */
-	public function document($documentGuid)
+	public function documentDetails($documentGuid)
```

Comment:
Sure, I can do that.  I mainly just changed it because the example that's on the README calls documentDetails so I thought this was a mistake.